### PR TITLE
Add template to run instrumented tests

### PR DESCRIPTION
**Pull request by p3dr0rv**

### CODE REVIEW

File: azure-pipelines/instrumented-tests/run-instrumented-tests.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# File: azure-pipelines\instrumented-tests\run-instrumented-tests.yml
```
Comment (iamgusain) [inline azure-pipelines/instrumented-tests/run-instrumented-tests.yml:1]: can we add it under azure-pipelines\templates\

File: azure-pipelines/instrumented-tests/run-instrumented-tests.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+# Description: Template to run Instrumented test.
+
+parameters:
+# Name of the git project we want to checkout.
+- name: gitProject
+  default: self
```
Comment (iamgusain) [inline azure-pipelines/instrumented-tests/run-instrumented-tests.yml:7]: you can add supported values as list here.

**Review by iamgusain — APPROVED** [review]

File: azure-pipelines/instrumented-tests/run-instrumented-tests.yml (hunk 1 above)
Comment (p3dr0rv) [reply azure-pipelines/instrumented-tests/run-instrumented-tests.yml:1]: we do not have templates folder here, but I can add it.

File: azure-pipelines/instrumented-tests/run-instrumented-tests.yml (hunk 2 above)
Comment (p3dr0rv) [reply azure-pipelines/instrumented-tests/run-instrumented-tests.yml:7]: This name can variate a lot, so maintaining a list will be hard.
The caller of the template can opt for not giving an alias to the repository source.

File: azure-pipelines/instrumented-tests/run-instrumented-tests.yml (hunk 1 above)
Comment (p3dr0rv) [reply azure-pipelines/instrumented-tests/run-instrumented-tests.yml]: https://github.com/AzureAD/android-complete/pull/190/commits/af8f3c603e23fcffcde4c81de9a2de8524d4ebde

**Pull request merged**